ajv_CXX_FLAG macro is broken and causes m4 to loop forever, let's just assume g++ supports
these flags, see http://bugs.gentoo.org/305305

--- configure.ac
+++ configure.ac
@@ -81,13 +81,9 @@
 
 
 ############################################################################
-# Set some compilter flags if they're available                            #
+# Set some compilter flags                                                 #
 ############################################################################
-ajv_CXX_FLAG(-g)
-ajv_CXX_FLAG(-Wall)
-ajv_CXX_FLAG(-Wno-unused-function)
-ajv_CXX_FLAG(-Wno-unused-variable)
-ajv_CXX_FLAG(-Wno-unused)
+CXXFLAGS="$CXXFLAGS -g -Wall -Wno-unused-function -Wno-unused-variable -Wno-unused"
 
 
 
